Oryx.Cognite/src/Transformations.fs

```diff
@@ -0,0 +1,209 @@
+        |> HttpHandler.create items (Url +/ "schedules")
+
+    /// Delete schedules for a list of transformations.
+    let unschedule (items: TransformationScheduleDelete) (source: HttpHandler<unit>) : HttpHandler<EmptyResponse> =
+        source
+        |> withLogMessage "Transformations:unschedule"
+        |> HttpHandler.delete items (Url +/ "schedules")
+
+    /// Update schedules for a list of transformations.
+    let updateSchedules
+        (items: UpdateItem<TransformationScheduleUpdate> seq)
+        (source: HttpHandler<unit>)
+        : HttpHandler<TransformationSchedule seq> =
+        source
+        |> withLogMessage "Transformations:updateSchedules"
+        |> HttpHandler.update items (Url +/ "schedules")
+
+    /// List notifications, optionally restricted to a single transformation, with pagination.
+    let listNotifications
+        (query: TransformationNotificationQuery)
+        (source: HttpHandler<unit>)
+        : HttpHandler<ItemsWithCursor<TransformationNotification>> =
+        source
+        |> withLogMessage "Transformations:listNotifications"
+        |> getWithQuery query (Url +/ "notifications")
+
+    /// Create notifications for transformations.
+    let subscribe
+        (items: TransformationNotificationCreate seq)
+        (source: HttpHandler<unit>)
+        : HttpHandler<TransformationNotification seq> =
+        source
+        |> withLogMessage "Transformations:subscribe"
+        |> HttpHandler.create items (Url +/ "notifications")
+
+    /// Delete notifications for transformations.
+    let deleteNotifications (items: int64 seq) (source: HttpHandler<unit>) : HttpHandler<EmptyResponse> =
+        let idts = items |> Seq.map Identity.Create
+        let req = ItemsWithoutCursor(Items = idts)
+
+        source
+        |> withLogMessage "Transformations:deleteNotifications"
+        |> HttpHandler.delete req (Url +/ "notifications")
+
+    /// Run a transformation query to preview the result.
+    let preview (query: TransformationPreview) (source: HttpHandler<unit>) : HttpHandler<TransformationPreviewResult> =
+        source
+        |> withLogMessage "Transformations:preview"
+        |> withCompletion HttpCompletionOption.ResponseHeadersRead
+        |> postV10 query (Url +/ "query/run")
+
+    /// Get the schema for a given transformation destination type and conflict mode.
+    let getSchema
+        (schemaType: TransformationDestinationType)
+        (conflictMode: TransformationConflictMode)
+        (source: HttpHandler<unit>)
+        : HttpHandler<IEnumerable<TransformationColumnType>> =
+        let query = TransformationSchemaQuery(ConflictMode = conflictMode)
+        let url = (Url +/ "schema" +/ schemaType.ToString())
+
+        http {
+            let! result =
+                source
+                |> withLogMessage "Transformations:schema"
+                |> getWithQuery<ItemsWithoutCursor<TransformationColumnType>> query url
+
+            result.Items
```

Comment:
Missing return?

Reply:
Huh. How is that even possible? Wouldn't the compiler catch that?

Reply:
I agree. Need to investigate why this was not discovered by the compiler